### PR TITLE
Exocom: use default port 80

### PR DESCRIPTION
**Pull request by hugobho**
resolves https://github.com/Originate/exosphere/issues/514

### CODE REVIEW

File: javascript/exorelay/src/exorelay.ls [hunk 1]
```diff
@@ -11,7 +11,7 @@ class ExoRelay extends EventEmitter
 
   (@config) ->
     @config?.exocom-host or throw new Error 'ExoCom host not provided to Exorelay'
-    @config?.exocom-port or throw new Error 'ExoCom port not provided to Exorelay'
+    @config.exocom-port? or @config.exocom-port = 80
```
Comment (charlierudolph) [inline javascript/exorelay/src/exorelay.ls]: Thoughts on update this function to start with
```
(@config = {}) ->
```
Then we can remove the `?` from the exocom-host and role as well.

Comment (charlierudolph) [reply javascript/exorelay/src/exorelay.ls]: I believe we can simplify this line to  `@config.exocom-port or= 80`

File: go/exoservice/exoservice.go [hunk 2]
```diff
@@ -32,17 +32,14 @@ type ReplyChannelMapping map[string]chan *structs.Message
 // Bootstrap brings an ExoService online using environment variables and the
 // given messageHandlers
 func Bootstrap(messageHandlers MessageHandlerMapping) {
-	port, err := strconv.Atoi(os.Getenv("EXOCOM_PORT"))
-	if err != nil {
-		panic(fmt.Sprintf("Invalid port: %s", os.Getenv("EXOCOM_PORT")))
-	}
+	port := getPort()
```
Comment (charlierudolph) [inline go/exoservice/exoservice.go]: Can this variable be removed and we can `getPort()` in the definition of `config`

Comment (hugobho) [reply go/exoservice/exoservice.go]: I did that at first but there's a print statement on line 46 that needs this var

Comment (charlierudolph) [reply go/exoservice/exoservice.go]: Danget, I looked for that and couldn't find it. Github did not expand the diff the way I expected. Good as it

**Comment by hugobho** [comment]
@charlierudolph comments addressed

**Pull request merged**